docs-partials/builder/common/BlockDevice.mdx

```diff
@@ -16,6 +16,7 @@ launch_block_device_mappings {
     encrypted = true
     kms_key_id = "1a2b3c4d-5e6f-1a2b-3c4d-5e6f1a2b3c4d"
 }
+
```

Comment:
I wonder why this changed, is it something that happens when you run `make generate`? If so we're good, I'm more curious than anything

Reply:
Yup! I didn't changed the file manually but with `make generate`.

Reply:
Fixed.